### PR TITLE
Use simplejson

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

File: synapse/handlers/federation.py [hunk 1]
```diff
@@ -324,14 +328,21 @@ def do_invite_join(self, target_hosts, room_id, joinee, content, snapshot):
...
+            logger.info("Content befhahs: %s", builder.content)
+
             add_hashes_and_signatures(
                 builder,
                 self.hs.hostname,
                 self.hs.config.signing_key[0],
             )
 
+            logger.info("Content aftet hah: %s", builder.content)
+            logger.info("Content pdu json: %s", builder.get_pdu_json())
```
Comment (NegativeMjark) [inline synapse/handlers/federation.py]: Might want to remove the debug logging

**Comment by NegativeMjark** [comment]
Might want to install simplejson on jenkins.

**Comment by NegativeMjark** [comment]
I've pip installed simplejson in the venv on jenkins.

**Comment by erikjohnston** [comment]
jenkins: test this please

**Comment by erikjohnston** [comment]
jenkins: test this please

**Pull request merged**